site/js/modules/webui/input.js

```diff
@@ -216,14 +216,14 @@ class FieldType {
         listOfFieldHTML.push(`<div class="error-message"><p><span></span></p></div>`);
 
         return `
-            <div class="input-field ${this.name} ${this.inputClasses}">
+            <div id="${this.name}" name="${this.name}" class="input-field ${this.inputClasses}">
```

Comment:
`name` is technically not a valid attribute on divs:
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes

Consider using a data attribute instead (`data-name`?).